cpp/arcticdb/storage/azure/azure_storage.cpp

```diff
@@ -350,6 +350,10 @@ AzureStorage::AzureStorage(const LibraryPath &library_path, OpenMode mode, const
             ARCTICDB_RUNTIME_DEBUG(log::storage(), "Using default CA cert path");
         else
             ARCTICDB_RUNTIME_DEBUG(log::storage(), "CA cert path: {}", conf.ca_cert_path());
+        if (conf.ca_cert_dir().empty())
```

Comment:
The un-bracketed ( `{}` ) `if-else` statements in this constructor are a bug waiting to happen aren't they, let's add brackets throughout this ctor while we're here

Reply:
Will amend.
Now I start appreciating the existence of code standard. I vaguely remember someone asked me to remove the bracket as it is unnecessary in the PR....